lightwood/api/ensemble.py

```diff
@@ -0,0 +1,64 @@
+from .predictor import Predictor
+from lightwood.constants.lightwood import ColumnDataTypes
+from collections import Counter
+import numpy as np
+import pickle
+import torch
+import os
+
+
+class LightwoodEnsemble:
+    def __init__(self, predictor=None, predictors=None, load_from_path=None):
```

Comment:
Can check `isinstance(predictors, list)` / `isinstance(predictors, Predictor)` instead of having two parameters